src/assets/styles/header.scss

```diff
@@ -47,9 +47,9 @@
   }
   button {
     width: 20%;
-    background: $dark-red;
+    background: $red;
```

Comment:
There are some color changes -- I realized I had misread the style guide and design re: default vs hover colors for buttons.